ts/components/messages/MessageDetail.tsx

```diff
@@ -119,3 +167,9 @@ export default class MessageDetailComponent extends React.PureComponent<Props> {
     );
   }
 }
+
+const mapStateToProps = (state: GlobalState) => ({
+  isDebugModeEnabled: state.debug.isDebugModeEnabled
```

Comment:
We are trying to have only screen connected to the store and pass props down.

Reply:
Fixed